examples/imagenet/imagenet_lib.py

```diff
@@ -191,58 +159,78 @@ def _prepare(x):
   return jax.tree_map(_prepare, xs)
 
 
-def create_input_iter(batch_size, image_size, dtype, train, cache):
-  ds = input_pipeline.load_split(
-      batch_size, image_size=image_size, dtype=dtype, train=train, cache=cache)
-  it = map(prepare_tf_data, ds)
+def _create_input_iter(dataset_builder, batch_size, image_size, dtype, train,
+                       cache):
+  ds = input_pipeline.create_split(
+      dataset_builder, batch_size, image_size=image_size, dtype=dtype,
+      train=train, cache=cache)
+  it = map(_prepare_tf_data, ds)
   it = jax_utils.prefetch_to_device(it, 2)
   return it
 
 
 # flax.struct.dataclass enables instances of this class to be passed into jax
 # transformations like tree_map and pmap.
 @flax.struct.dataclass
-class TrainState:
+class _TrainState:
```

Comment:
ditto on this and all the others, we don't need to add underscores to any of these

Reply:
done.